ci-scripts/compile-settings.jq

```diff
@@ -10,6 +12,12 @@ if $for_release then {
   features: ($matrix.debug_features // ["rustls", "fancy-with-backtrace"]),
 } end
 |
+.rustflags = (
+  if $for_release and $matrix.target == "aarch64-unknown-linux-musl" or $matrix.target == "armv7-unknown-linux-musleabihf"
+  then "-C link-arg=-lgcc -Clink-arg=-static-libgcc"
```

Comment:
I'd rather we have anything that's not default in the build.yml matrix, but can be refactored later